ginga/qtw/GingaQt.py

```diff
@@ -633,7 +633,11 @@ def new_ws_cb(self, w, rsp, b, names):
     def gui_load_file(self, initialdir=None):
         if self.filesel.exec_():
             fileNames = list(map(str, list(self.filesel.selectedFiles())))
-            self.load_file(fileNames[0])
+            lf = self.load_file(fileNames[0])
+
+        # Load MultiDim plugin if file is NDIM
```

Comment:
I think this needs to be indented to fall under the if.  Otherwise variable `lf` is not defined.  Suggest variable `image` instead of `lf`, as this would be consistent with other usage.